packages/integration-tests/src/api/application.ts

```diff
@@ -26,10 +26,7 @@ export const createApplication = async (
 
 export const getApplications = async (types?: ApplicationType[]) => {
   const searchParams = new URLSearchParams(
-    conditional(
-      types &&
-        types.length > 0 && [...types.map((type) => ['search.type', type]), ['mode.type', 'exact']]
-    )
+    conditional(types && types.length > 0 && types.map((type) => ['types', type]))
   );
```

Comment:
i assume the issue was that application search with `type` hasn't been covered by test cases. it looks like we are not adding api test cases to cover this situation

Reply:
In this PR, we added console test cases to ensure the functionality of the search component.
Also add more API test cases for search assigned entities with keyword.

Reply:
API tests and UI tests have no conflict though

Reply:
API tests added (test cases covered filtering the roles/applications by `type` and `search` parameters, which should be the same as the actual API calls triggering by the console when using the search component).